tasks/LaundrySort.tex

```diff
@@ -0,0 +1,44 @@
+\section{Laundry Sort [Housekeeper]}
+The robot is helping to do laundry by sorting it.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+The robot is given a bin of laundry and must sort it by color and size. There are both children's and adult's clothing in the bin, and the robot must make piles by color and size. There are existing piles of laundry on the floor into which the laundry in the bin must be sorted.
+
+\noindent\textbf{Reward:} 2500pts (400pts per object).
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Identifying that the bin is empty (250pts)
+	\item Moving the empty bin to the bedroom (250pts)
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}[nosep]
+	\item \textbf{Laundry bin:} Place clothes in bin and bin in living room.
+
+	\item \textbf{Laundry piles:} Place piles of clothing onto the floor near the bin sorted by color and size.
```

Comment:
Sorting by colour and size, although I get what's it's trying to do is hard to organize if we're going to ask for laundry on site... if this was a defined set it might make it more clear how things are to be sorted. Are we sorting Whites, Blacks, and all other colours?